modules/system/ServiceProvider.php

```diff
@@ -575,6 +576,15 @@ protected function registerValidator()
             $validator->replacer('extensions', function ($message, $attribute, $rule, $parameters) {
                 return strtr($message, [':values' => implode(', ', $parameters)]);
             });
+
+            $plugins = PluginManager::instance()->getRegistrationMethodValues('registerValidator');
```

Comment:
```suggestion
            $plugins = PluginManager::instance()->getRegistrationMethodValues('registerValidators');
```
Also needs PR to docs repo

Reply:
Added PR to docs repo to support this PR.